pkg/build/builder/sti.go

```diff
@@ -227,7 +227,7 @@ func (s *S2IBuilder) Build() error {
 	config.PullAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(config.BuilderImage, dockercfg.PullAuthType)
 	config.IncrementalAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(pushTag, dockercfg.PushAuthType)
 
-	glog.V(0).Infof("Creating a new S2I builder with build config: %#v\n", describe.DescribeConfig(config))
+	glog.V(4).Infof("Creating a new S2I builder with build config: %#v\n", describe.DescribeConfig(config))
```

Comment:
Printing information on failures that lets you pin down the user problem is what we need to be doing.  Dumping the JSON doesn't help a user - printing real errors does.